README.md

```diff
@@ -0,0 +1,37 @@
+Successor of Open-ZaakBrug, an app for Dutch municipalities that supports the transition from "zaak- en documentatieservices" (zds) to "zaakgericht werken" (zgw).
+
+Dutch municipalities provide services that are delivered many times. An example of such a service is to provide passports. This service is delivered many times because each citizen needs a passport or an ID card. An instance of a service that is provided multiple times is referred by the Dutch word "zaak" (case).
+
+There is a new specification named "zaakgericht werken" (zgw) for case register systems. Zgw is part of the Common Ground specification for software used by Dutch municipalities (https://vng.nl/artikelen/common-ground). Zgw will replace an old specification called "zaak- en documentatieservices" (zds). The municipality of Súdwest Fryslân wants to migrate to the zgw standard without having to invest in zds interfaces. To this end, the municipality has built the system Open-ZaakBrug (brug means bridge). It acts as an interface between a zgw case register system on the one hand and zds systems connecting to the case register system on the other hand. Open-ZaakBrug is hosted at https://github.com/Sudwest-Fryslan/OpenZaakBrug. For a comparison of zgw and zds see https://vng.nl/sites/default/files/2022-03/20200703%20-%20Vergelijking%20ZGW%20APIs%20en%20ZDS_v1.0.pdf#:~:text=De%20Zaak-%20en%20Documentservices%20%28ZDS%29-standaard%20beschrijft%20hoe%20een,vakapplicatie%20en%20deze%20worden%20gekopieerd%20naar%20het%20zaaksysteem.
```

Comment:
I think it would look nicer to have the links as text 

There is a new specification named "zaakgericht werken" (zgw) for case register systems. Zgw is part of the [Common Ground specification](https://vng.nl/artikelen/common-ground) for software used by Dutch municipalities. Zgw will replace an old specification called "zaak- en documentatieservices" (zds). The municipality of Súdwest Fryslân wants to migrate to the zgw standard without having to invest in zds interfaces. To this end, the municipality has built the system OpenZaakBrug (brug means bridge). It acts as an interface between a zgw case register system on the one hand and zds systems connecting to the case register system on the other hand. The OpenZaakBrug source code is hosted on [GitHub Sudwest-Fryslan/OpenZaakBrug](https://github.com/Sudwest-Fryslan/OpenZaakBrug). For a comparison of zgw and zds see [API-standaarden voor Zaakgericht Werken vergeleken met de Zaak- en Documentservices](https://vng.nl/sites/default/files/2022-03/20200703%20-%20Vergelijking%20ZGW%20APIs%20en%20ZDS_v1.0.pdf#:~:text=De%20Zaak-%20en%20Documentservices%20%28ZDS%29-standaard%20beschrijft%20hoe%20een,vakapplicatie%20en%20deze%20worden%20gekopieerd%20naar%20het%20zaaksysteem).

```suggestion
There is a new specification named "zaakgericht werken" (zgw) for case register systems. Zgw is part of the [Common Ground specification](https://vng.nl/artikelen/common-ground) for software used by Dutch municipalities. Zgw will replace an old specification called "zaak- en documentatieservices" (zds). The municipality of Súdwest Fryslân wants to migrate to the zgw standard without having to invest in zds interfaces. To this end, the municipality has built the system OpenZaakBrug (brug means bridge). It acts as an interface between a zgw case register system on the one hand and zds systems connecting to the case register system on the other hand. The OpenZaakBrug source code is hosted on [GitHub Sudwest-Fryslan/OpenZaakBrug](https://github.com/Sudwest-Fryslan/OpenZaakBrug). For a comparison of zgw and zds see [API-standaarden voor Zaakgericht Werken vergeleken met de Zaak- en Documentservices](https://vng.nl/sites/default/files/2022-03/20200703%20-%20Vergelijking%20ZGW%20APIs%20en%20ZDS_v1.0.pdf#:~:text=De%20Zaak-%20en%20Documentservices%20%28ZDS%29-standaard%20beschrijft%20hoe%20een,vakapplicatie%20en%20deze%20worden%20gekopieerd%20naar%20het%20zaaksysteem).
```